docs/tutorials/services/navigate-with-rover-base.md

```diff
@@ -36,12 +36,21 @@ weight: 5
 # SMEs: Ray Bjorkman, Fahmina
 ---
 
-One key feature of Viam is [the navigation service](/mobility/navigation/), the stateful definition of Viam's [motion service](/mobility/motion/).
+The [navigation service](/mobility/navigation/) is the stateful definition of the [motion service](/mobility/motion/).
```

Comment:
```suggestion
```
I think this can just go. The next sentences describe it

Reply:
I see what you mean but I think it's awk as a first sentence--made a suggestion